beep/protocol/maccor_to_biologic_mb.py

```diff
@@ -0,0 +1,301 @@
+# Copyright 2019 Toyota Research Institute. All rights reserved.
+""" Parsing and conversion of maccor procedure files to arbin schedule files"""
+
+import os
+import re
+import xmltodict
+
+class MaccorToBiologicMb:
+    """
+    Collection of methods to convert maccor protocol files to biologic modulo bat protocol files
+    Differences in underlying hardware and software makes this impossible in some cases
+    or forces workarounds that may fail. Certain work-arounds may be attempted and are
+    documented under the conversion function.
+    """
+
+    def __init__(self):
+        self.blank_seq = {
+            # sequence number
+            "Ns": "0",
```

Comment:
We might want to move this information into a .yaml file. See /beep/beep/protocol/protocol_schemas/arbin_schedule_schema.yaml

Reply:
Ah, sure.

As a personal preference I prefer to keep as much context in a single file as possible to reduce cognitive load from switching between files but we should be consistant, will move this to a file named `biologic_modulo_bat_schema.yaml`